### PR TITLE
Setup Ionic Test Environment

### PR DESCRIPTION
**Pull request by AKidd95**
#### Description

* Setup testing environment for Karma and Protractor
* Create acceptance and unit test files
* Edit versions of conflicting dependencies

#### What I have learned working on this feature: 
* How to properly setup a testing environment for ionic
* Basic commands for running unit and acceptance tests

### CODE REVIEW

**Review by isabellelii — APPROVED** [review]
Looks all good to me!

File: e2e/app.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  beforeEach(() => {
+    page = new Page();
+  });
+
+  describe('default screen', () => {
+    beforeEach(() => {
+      page.navigateTo('/');
+    });
+  });
+});
```
Comment (SophieSjoberg) [inline e2e/app.e2e-spec.ts]: You should also test for page title here :)

**Review by SophieSjoberg — CHANGES_REQUESTED** [review]

File: e2e/app.e2e-spec.ts (hunk 1 above)
Comment (AKidd95) [reply e2e/app.e2e-spec.ts]: oh ok, we thought that was just an example test, we will make sure to do that

Comment (diraulo) [reply e2e/app.e2e-spec.ts]: Has this been done? any plans on doing that on this PR? or a separate one?

**Review by SophieSjoberg — APPROVED** [review]

**Pull request merged**